tests/nosetests/pyanaconda_tests/module_storage_test.py

```diff
@@ -856,3 +932,128 @@ def format_test(self, blockdev):
         ])
+
+
+class ManualPartitioningInterfaceTestCase(unittest.TestCase):
+    """Test DBus interface of the manual partitioning module."""
+
+    def setUp(self):
+        """Set up the module."""
+        self.manual_part_module = ManualPartitioningModule()
+        self.manual_part_interface = ManualPartitioningInterface(self.manual_part_module)
+
+    def _test_dbus_property(self, *args, **kwargs):
+        check_dbus_property(
+            self,
+            MANUAL_PARTITIONING,
+            self.manual_part_interface,
+            *args, **kwargs
+        )
+
+    def enabled_property_test(self):
+        """Test the enabled property."""
+        self._test_dbus_property(
+            "Enabled",
+            True
+        )
+
+        self._test_dbus_property(
+            "Enabled",
+            False
+        )
+
+    def mount_points_property_test(self):
+        """Test the mount points property."""
+        self._test_dbus_property(
+            "MountPoints",
+            []
+        )
+
+        in_value = [
+            {
+                MOUNT_POINT_PATH: "/boot",
+                MOUNT_POINT_DEVICE: "/dev/sda1"
+            }
+        ]
+
+        out_value = [
+            {
+                MOUNT_POINT_PATH: get_variant(Str, "/boot"),
+                MOUNT_POINT_DEVICE: get_variant(Str, "/dev/sda1"),
+                MOUNT_POINT_REFORMAT: get_variant(Bool, False),
+                MOUNT_POINT_FORMAT: get_variant(Str, ""),
+                MOUNT_POINT_FORMAT_OPTIONS: get_variant(Str, ""),
+                MOUNT_POINT_MOUNT_OPTIONS: get_variant(Str, "")
+            }
+        ]
+
+        self._test_dbus_property(
+            "MountPoints",
+            in_value,
+            out_value
+        )
+
+        in_value = [
+            {
+                MOUNT_POINT_PATH:  "/boot",
+                MOUNT_POINT_DEVICE: "/dev/sda1",
+                MOUNT_POINT_REFORMAT: True,
+                MOUNT_POINT_FORMAT: "xfs",
+                MOUNT_POINT_FORMAT_OPTIONS: "-L BOOT",
+                MOUNT_POINT_MOUNT_OPTIONS: "user"
+            }
+        ]
+
+        out_value = [
+            {
+                MOUNT_POINT_PATH: get_variant(Str, "/boot"),
+                MOUNT_POINT_DEVICE: get_variant(Str, "/dev/sda1"),
+                MOUNT_POINT_REFORMAT: get_variant(Bool, True),
+                MOUNT_POINT_FORMAT: get_variant(Str, "xfs"),
+                MOUNT_POINT_FORMAT_OPTIONS: get_variant(Str, "-L BOOT"),
+                MOUNT_POINT_MOUNT_OPTIONS: get_variant(Str, "user")
+            }
+        ]
+
+        self._test_dbus_property(
+            "MountPoints",
+            in_value,
+            out_value,
+        )
+
+        in_value = [
+            {
+                MOUNT_POINT_PATH: "/boot",
+                MOUNT_POINT_DEVICE: "/dev/sda1"
+            },
+            {
+                MOUNT_POINT_PATH: "/",
+                MOUNT_POINT_DEVICE: "/dev/sda2",
+                MOUNT_POINT_REFORMAT: True
```

Comment:
I wouldn't use constants in the `in_value`. If a typo will happen in the constant it won't be found this way.

Reply:
Fixed.